src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.tasks.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a TaskList class that is responsible for managing the tasks.
+ * 
+ * @author Ramanathan Kumarappan
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
+
+    /**
+     * Constructor for TaskList.
+     */
+    TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Add Task to the TaskList.
+     * @param t - The Task to be added.
+     */
+    public void addTask(Task t) {
+        this.tasks.add(t);
+    }
+
+    /**
+     * Retrieves the specified Task from TaskList.
+     * @param i - An int specifying the index of the Task in the TaskList.
+     * @return The Task at the given index.
+     */
```

Comment:
I don't think I like that you added a dash ("-") before writing the description for each parameter. I have seen this in other places as well.

To add, I realised some inconsistency in adding dashes in return statements, e.g. in this case you didn't, but you did at getUserCommand() of Ui.java.